docs/en/observability/apm/shared-kibana-endpoint.asciidoc

```diff
@@ -1,6 +1,7 @@
 // tag::shared-kibana-config[]
-APM Server uses the APM integration to set up and manage APM templates, policies, and pipelines.
-To confirm the integration is installed, APM Server polls either {es} or {kib} on startup.
+APM Server uses the {es} apm-data plugin to set up and manage APM templates, policies, and pipelines.
+// Does it make sense to say the plugin is "installed"?
+To confirm the plugin is installed, APM Server polls either {es} or {kib} on startup.
```

Comment:
```suggestion
```
This statement is no longer correct. apm server 8.15+ operates with the assumption that the plugin is present, and there is no longer a polling step.